distribute.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'signing'
 sourceCompatibility = 1.8
 
 group = 'com.flightstats'
-version = '0.1.12-SNAPSHOT'
+version = '0.1.12'
```

Comment:
This is dangerous for current consumers of 0.1.12-SNAPSHOT (I have code which does at the moment), since the snapshot versions will instantly be removed from maven central as soon as this one is published.. I suggest we either adopt a convention with even numbers being snapshots and odd numbers be releases, or add a 'b' to the snapshot versioning.